src/server/index.ts

```diff
@@ -5,9 +5,16 @@ import bodyParser from 'body-parser'
 
 import config from './config'
 import api from './api'
-import CheckerController from './checker'
+import { initSequelize } from './models'
+import { CheckerController, CheckerService } from './checker'
 
-const checker = new CheckerController()
+const { models } = initSequelize()
```

Comment:
Should we wait for the promise returned by `sequelize.sync()` to resolve before starting the express server?

Reply:
Good catch! I'll change it once #10 is merged